content/javascript/concepts/dates/terms/setMinutes/setMinutes.md

```diff
@@ -0,0 +1,43 @@
+---
+Title: '.setMinutes()'
+Description: 'Changes the minute value of the Date object according to local time.'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Date'
+  - 'Methods'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`.setDate()`** method changes the minute value of the Date object according to local time.
+
+## Syntax
+
+```javascript
+myDate.setMinutes(minutesValue);
+```
+
+The **`.setMinutes()`** method is called on `Date` object `myDate` with an input argument `minutesValue` that contains the desired minute value.
+
+## Example
+
+```javascript
```

Comment:
The current example is the same as the codebyte. I'd leave the codebyte as-is and update the example to use a specific date value, so we can log it to the console and show an output block with a constant value.

Reply:
Hi, I changed the example so it uses a static date. Please see commits `7ef3b5e35e77abc92213713aca21536d11fc6870` and `520bd0a1db7f0813ae028222346a6e66cf8aeb45`.